src/defs/shared.ts

```diff
@@ -1336,15 +1507,6 @@ tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Shared;
 SharedDefinitions.push(tmpDef);
 
-tmpDef = new MTAFunction;
-tmpDef.label = "getPedSkin";
```

Comment:
Shouldn't this say "DEPRECATED" instead of removing it completely? Also add a section for deprecated functions?